Resources/Prototypes/_NF/Shipyard/bulkermax.yml

```diff
@@ -0,0 +1,28 @@
+- type: vessel
+  id: BULKER
+  name: NC Bulker
+  description: Small mining vessel for deep space operations
+  price: 35000 #31000ish after the mapinit appraisal
```

Comment:
Keep in mind ships wont pass new price testing unless they are more then 15% added